seed/challenges/02-javascript-algorithms-and-data-structures/basic-data-structures.json

```diff
@@ -5,17 +5,58 @@
   "helpRoom": "Help",
   "challenges": [
     {
+      "id": "587d7dac367417b2b2516zx5",
+      "title": "Basic Data Structures: Introduction to Arrays",
```

Comment:
I think this can simply be "Introduction to Arrays" since the heading is above. (see: HTML section)